lib/services/database.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class DatabaseMethods {
+  getUserByUserName(String username) async {
```

Comment:
what type are you returning here? try to avoid dynamic typing and return strong typed data